test/functional/test_python_usage.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from taskotron_python_versions.python_usage import file_contains
+from taskotron_python_versions.common import file_contains
```

Comment:
Maybe the test should move to `test_common.py`?

Reply:
Hm, we do not yet have `test_common.py`, so I will just rename this file.

Reply:
Makes sense if it only tests that function. Thanks